### DISCUSSION TITLE
Planning poker issue for guest users

### DISCUSSION THREAD

**Discussion by anthonyroux** [Carduelis/poker-widget-preact#8] [Q&A]
Hi,

I tested planning poker with someone who doesn't have a Miro account and I am facing an issue.
You see it on this [Loom video](https://www.loom.com/share/e11ef68cf0a74008be6c50337445275a).

To reproduce:
1. I have a Miro account (Enterprise plan), I create. a new board and configure the sharing policy to "anyone with a link can edit" (you can access the board [here](https://miro.com/app/board/uXjVOM-aTpw=/?invite_link_id=771527063056))
2. Planning poker works fine for me on this board
3. I share the link with someone (or just open a new browser in incognito mode)
4. I try to use Planning poker, the app opens well but when I try to drag and drop a card it doesn't work (see video)

**Comment by Carduelis** [comment]
@anthonyroux hey! Thank you for the feedback. We will look into it this week!

**Reply by Luab** [reply]
Hi. Are there any updates on this issue? The app is absolutely fabulous and helps our team greatly. However, we have some on and off collaborators that are not joining our team and get access to specific boards instead.

**Reply by ngalarza10** [reply]
@Carduelis also looking to see if there is an update on this. Guest users received a multitude of network errors

**Reply by Carduelis** [reply]
@Luab , @ngalarza10 We contacted the Miro team and they assured us that they are working on this feature.
I'll ping here once the issue is resolved.
